extraction/extractor.py

```diff
@@ -48,19 +49,19 @@ def extract_html(self):
         try:
             # To request Html from URL
             file = REQUEST_SESSION.get(self.url).text
-        except requests.exceptions.ConnectionError as err:
-            LOGGER.error(err)
+        except requests.exceptions.ConnectionError:
+            raise WebsiteConnectionError(self.url)
 
         # Remove the comment to read local files for testing purpose
         # file = open('./test_html.html','r').read()
 
-        if file:
-            self.soup = BeautifulSoup(file, 'lxml')
-            self.clean_soup()
+        if not file:
+            raise NoneTypeMarkupError
```

Comment:
missing brackets maybe???
```suggestion
            raise NoneTypeMarkupError()
```

Reply:
My bad! I will update it.